torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -190,6 +191,15 @@ std::string valueName(Value * n) {
   return "n" + std::to_string(n->unique());
 }
 
+std::string valueNameOrConstant(Value * n) {
```

Comment:
What is going on here, I don't see where constant_as needs to be injected here.

Reply:
Hmmm yeah I think I went a bit too far here. I wanted to avoid a situation where this:
```
%1 = prim::Constant[value={0}]()
%2 = aten::add(%1, %0)
```
causes the `FusionGroup` to get `%1` as an input, when we could simply inline it into the kernel. This wasn't a problem previously, because we used to put constants in the `other` attribute. On the other hand this will probably trigger an error if the constant is not a scalar, so I need to fix this.